doc/api/assert.markdown

```diff
@@ -12,14 +12,9 @@ The API for the `assert` module is [Locked][]. This means that there will be no
 additions or changes to any of the methods implemented and exposed by
 the module.
 
-## assert(value[, message]), assert.ok(value[, message])
+## assert(value[, message])
 
-Tests if `value` is truthy. It is equivalent to
-`assert.equal(!!value, true, message)`.
-
-If `value` is not truthy, an `AssertionError` is thrown with a `message`
-property set equal to the value of the `message` parameter. If the `message`
-parameter is `undefined`, a default error message is assigned.
+An alias of [`assert.ok()`][] .
```

Comment:
This isn't true. See e.g. https://github.com/nodejs/node/blob/master/doc/api/assert.markdown#assertdoesnotthrowblock-error-message